### PR TITLE
[feat] google maps integrated 

### PR DESCRIPTION
**Pull request by taleymirza**
Issue #96 

- with a basic maps functionality with static location
- will have refactoring and improvements in next commit

### CODE REVIEW

**Comment by taleymirza** [comment]
I will continue it on monday

File: client/src/components/GoogleMap/GoogleMap.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+class GoogleMap extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showingInfoWindow: false,
+      activeMarker: {},
+      selectedPlace: {},
+    };
+    // binding this to event-handler functions
+    this.onMarkerClick = this.onMarkerClick.bind(this);
```
Comment (mashhoodr) [inline client/src/components/GoogleMap/GoogleMap.js]: @taleymirza we dot not use this convention anymore, if we have arrow functions then we dont need this

Comment (taleymirza) [reply client/src/components/GoogleMap/GoogleMap.js]: Yup, I use this code from a resource to test map, I will refactor it

**Comment by taleymirza** [comment]
@mashhoodr I have created issue #129 for this..

**Comment by mashhoodr** [comment]
@taleymirza please fix codacy issues

**Pull request merged**